omod/src/main/webapp/resources/htmlforms/PatientReferral.xml

```diff
@@ -182,6 +222,20 @@
                 });
             });
 
+            function getPatientTransferEncounterTypes(encounterType) {
```

Comment:
@dbaluku move this to the ugandaemr.js file so that we can reuse it elsewhere

Reply:
Done